templates/docs/patterns/images.md

```diff
@@ -30,6 +30,14 @@ When an image needs a caption, it can be wrapped in a `<figure>` element, along
 View example of image with a caption
 </a></div>
 
+## Media element with spacing
+
+Adds spacing to the top of a media element.
```

Comment:
```suggestion
Adds spacing to the top of a media element. This is usually needed when image or video is placed below the horizontal rule component to avoid media touching the line.
```